lm_eval/tasks/kmmlu/utils.py

```diff
@@ -0,0 +1,18 @@
+import datasets
+
+def process_docs(dataset: datasets.Dataset) -> datasets.Dataset:
+    def _process_doc(doc):
+        instruction = f"""다음을 읽고 정답으로 알맞은 것을 고르시요.
+            ### Context: {doc["context"]}
+            ### Question: {doc["question"]}
+            ### Options:
+            (1) {doc['option#1']}\n(2) {doc["option#2"]}\n(3) {doc["option#3"]}\n(4) {doc['option#4']}}
+            ### Answer: 주어진 문제의 정답은"""
+        out_doc = {
+            "question": instruction,
+            "choices": ["(1)", "(2)", "(3)", "(4)"],
+            "gold": int(doc["gold"]) - 1,
+        }
+        return out_doc
+
+    return dataset.map(_process_doc)
```

Comment:
There seems to be an error in the formatting, try `pre-commit run --all-files` or chunk each instruction line into individual f-string lines.

Reply:
Thanks for the review! I've included a fix for this bug in commit 7bc9b82bbfc1f8293518d24a648002bd484985e3.